src/App/App.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faUser } from '@fortawesome/free-regular-svg-icons';
+import {
+  AuthorizationGrantType,
+  ConnectedOauthCallback,
+  getOpenSRPUserInfo,
+  useOAuthLogin,
+} from '@onaio/gatekeeper';
+import ConnectedPrivateRoute from '@onaio/connected-private-route';
+import { Helmet } from 'react-helmet';
+import { Layout } from 'antd';
+import { Switch, Route, Redirect } from 'react-router';
+import Loading from '../components/page/Loading';
+import { CustomLogout } from '../components/Logout';
+import { WEBSITE_NAME, BACKEND_ACTIVE } from '../configs/env';
+import {
+  REACT_CALLBACK_PATH,
+  BACKEND_CALLBACK_URL,
+  BACKEND_LOGIN_URL,
+  BACKEND_CALLBACK_PATH,
+} from '../constants';
+import { providers } from '../configs/settings';
+import ConnectedHeader from '../containers/ConnectedHeader';
+import Home from '../containers/pages/Home/Home';
+import CustomConnectedAPICallBack from '../components/page/CustomCallback';
+import './App.css';
+
+const { Content } = Layout;
+
+library.add(faUser);
+
+const App = (): JSX.Element => {
```

Comment:
I think the more generally accepted way to write this type definition is something like this:

```ts
const Component: React.FC<Props> = (props) => (
```
or where there are no props

```ts
const Component: React.FC = () => (
```
```